pom.xml

```diff
@@ -122,7 +122,7 @@
         <xpp3.servicemix.version>1.1.4c_7</xpp3.servicemix.version>
         <kxml2.servicemix.version>2.3.0_3</kxml2.servicemix.version>
         <!-- double-check downstream projects before changing jackson version -->
-        <fasterxml.jackson.version>2.9.8</fasterxml.jackson.version> <!-- To match cxf-jackson (from cxf-jaxrs) -->
+        <fasterxml.jackson.version>2.10.0.pr2</fasterxml.jackson.version> <!-- 2.9.9 matches cxf-jackson (from cxf-jaxrs), but 2.10 has better yaml support -->
```

Comment:
What better yaml support does 2.10 give?

Do we really need to upgrade `fasterxml.jackson.version` and `snakeyaml.version`? Things are simpler when our versions match the versions of things pre-shipped in karaf, or included in low-level features such as cxf.

It reduces the risk of weird startup problems where us loading a newer version causes lots of low-level bundles to be refreshed/rewired. Those startup problems can cause Karaf to fail to start if they are too slow.

CXF 3.3.2 (which is the version that goes with Karaf 4.2.7) ships with fasterxml 2.9.8 and with snakeyaml 1.24.